domains/certificates/Query.API/API/GranularCertificateIssuer/WalletException.cs

```diff
@@ -1,16 +1,10 @@
 using System;
-using System.Runtime.Serialization;
 
 namespace API.GranularCertificateIssuer;
 
-[Serializable]
```

Comment:
Why has this been removed?

Reply:
A mistake. Added it back: https://github.com/Energinet-DataHub/energy-origin/pull/521/commits/8b82fef0827618c5f917f6e7e6e082a3885fb9a4